docs/Guide/preconditions/command-cooldown.mdx

```diff
@@ -1,6 +1,278 @@
+<TabItem value="javascript" label="JavaScript" default>
+
+```javascript {7}
+const { Command } = require('@sapphire/framework');
+const { Time } = require('@sapphire/time-utilities');
+
+module.exports = class PingCommand extends Command {
+  constructor(context) {
+    super(context, {
+      // ...
+      cooldownDelay: Time.Second * 10; // Much easier for humans to parse
+    });
+  }
+};
+```
+
+</TabItem>
+<TabItem value="ESM" label="ESM">
+
+```javascript {7}
+import { Command } from '@sapphire/framework';
+import { Time } from '@sapphire/time-utilities';
+
+export class PingCommand extends Command {
+  constructor(context) {
+    super(context, {
+      // ...
+      cooldownDelay: Time.Second * 10; // Much easier for humans to parse
+    });
+  }
+}
+```
+
+</TabItem>
+<TabItem value="typescript" label="TypeScript">
+
+```typescript {6}
+import { ApplyOptions } from '@sapphire/decorators';
+import { Command, CommandOptions } from '@sapphire/framework';
+import { Time } from '@sapphire/time-utilities';
+
+@ApplyOptions<CommandOptions>({
+  // ...
+  cooldownDelay: Time.Second * 10; // Much easier for humans to parse
+})
+export class PingCommand extends Command {}
+```
+
+</TabItem>
+</Tabs>
+
+You can view the docs [here][timeutils].
+
+:::
+
+## User Exceptions
+
+It's very common to not want cooldowns to apply to certain people, for example, the bot owner. This can be achieved by
+adding [`cooldownFilteredUsers`][cooldownfilteredusers] to the options. This option should be an array of users ID that
+the bot can ignore when calculating cooldowns.
+
+<Tabs groupId="language-choice">
+<TabItem value="javascript" label="JavaScript" default>
+
+```javascript {8}
+const { Command } = require('@sapphire/framework');
+
+module.exports = class PingCommand extends Command {
+  constructor(context) {
+    super(context, {
+      // ...
+      cooldownDelay: 10_000, // 10_000 milliseconds
+      cooldownFilteredUsers: ['YOUR_ID'] // Ignore the bot owner
+    });
+  }
+};
+```
+
+</TabItem>
+<TabItem value="ESM" label="ESM">
+
+```javascript {8}
+import { Command } from '@sapphire/framework';
+
+export class PingCommand extends Command {
+  constructor(context) {
+    super(context, {
+      // ...
+      cooldownDelay: 10_000, // 10_000 milliseconds
+      cooldownFilteredUsers: ['YOUR_ID'] // Ignore the bot owner
+    });
+  }
+}
+```
+
+</TabItem>
+<TabItem value="typescript" label="TypeScript">
+
+```typescript {7}
+import { ApplyOptions } from '@sapphire/decorators';
+import { Command, CommandOptions } from '@sapphire/framework';
+
+@ApplyOptions<CommandOptions>({
+  // ...
+  cooldownDelay: 10_000 // 10_000 milliseconds
+  cooldownFilteredUsers: ['YOUR_ID'] // Ignore the bot owner
+})
+export class PingCommand extends Command {}
+```
+
+</TabItem>
+</Tabs>
+
+## Advanced Usage
+
+Accompanying `cooldownDelay`, you also have access to the options [`cooldownLimit`][cooldownlimit] and
+[`cooldownScope`][cooldownscope].
```

Comment:
![image](https://user-images.githubusercontent.com/4019718/143722762-63324b1e-20ee-4d92-8ddc-fc7dbb601e4c.png)

Reply:
Is [this](https://github.com/sapphiredev/website/pull/91/files#r757819950) resolved?

Reply:
yes

![image](https://user-images.githubusercontent.com/4019718/143785678-01759432-bd50-48a2-af74-0bc8b814de0d.png)